.github/workflows/compatibility.yml

```diff
@@ -34,27 +28,26 @@ jobs:
 
     - name: Setup Elixir and Erlang versions
       uses: erlef/setup-beam@v1
+      id: beam
       with:
-        otp-version: ${{ matrix.beam.otp }}
-        elixir-version: ${{ matrix.beam.elixir }}
+        version-file: .tool-versions
+        version-type: strict
 
```

Comment:
The compatibility workflow should check a range of old/new Elixir and Erlang/OTP versions rather than use the main `.tool-versions`. Removing credo and updating to the new actions v3 versions looks good though

Reply:
Right, my understanding is that it is valuable to check against a range of Event Store DB versions, but since Elixir 1.10, there is not much value in adding such a matrix IMO.

Being said, sure I can add it back